contracts/interchain-token-service/src/msg.rs

```diff
@@ -127,6 +127,7 @@ pub struct ChainConfig {
     pub chain: ChainNameRaw,
     pub its_edge_contract: Address,
     pub truncation: TruncationConfig,
+    pub translation_contract: Address,
```

Comment:
```suggestion
    pub msg_translator_contract: Addr,
```

Reply:
Should it be `Addr` since it's an axelar address? Or because we need to additional validation?